vagrant/provision/setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Capture network variable:
+NETWORK_BASE=$1
+sudo -i
+
+echo "Check eth1 ip config for netbase: $NETWORK_BASE"
+
+if ip a show dev eth1 | grep -q "inet $NETWORK_BASE"; then
+  echo "eth1 ip detected"
+else
+  echo "eth1 missing ip; restaring interface"
+  ifdown eth1 && ifup eth1
+fi
+
+
+echo "Install ntp server to avoid to have desync issues"
+yum -y install ntp 
+systemctl enable ntpd
+systemctl start ntpd
```

Comment:
I think you are right. I was looking at this issue https://github.com/openshift/openshift-ansible-contrib/issues/96 but as you mentioned ansible already handles [this](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_clock/tasks/main.yaml). 

Since the ntp portion is unnecessary I'll remove it from the pull request; the eth1 vagrant issue is legit.